sandbox/tests/group/group.ts

```diff
@@ -40,25 +42,12 @@ for (var i = 0; i < numActors; i++) {
       this.vel.y = -1 * Math.abs(this.vel.y);
     }
   });
-  actor.on('precollision', function() {
-    //console.log('inner collision');
-  });
 
   actor.vel.x = ex.Util.randomInRange(minVel, maxVel);
   actor.vel.y = ex.Util.randomInRange(minVel, maxVel);
-
-  blockGroup.add(actor);
+  engine.add(actor);
```

Comment:
Should we not keep the visual indication of collision by flipping colors?

Reply:
Would you like the blocks to change color, maybe to a teal-ish color to match the block?

Reply:
![collisiongroups](https://user-images.githubusercontent.com/612071/58604913-aeee5300-825b-11e9-949a-61239e36846b.gif)